handlers/data/handler.go

```diff
@@ -83,7 +86,7 @@ func Handler(w http.ResponseWriter, req *http.Request) {
 
 		stubData = &models.Dataset{
 			ID:         "AF001EW",
-			Title:      "AF001EW  Members of the Armed Forces by residence type by sex by age",
```

Comment:
I am personally unhappy with removing title field from the stub even if we don't have a way to use that field in the real API for now I think we should keep it in the stub. We need to know what tis the `title` or a `name` of the dataset in order to be able to display that information to the user. It could be added manually I suppose or even be the file name but please do not drop it all together.